packages/components/src/Checkbox/Checkbox.module.css

```diff
@@ -0,0 +1,76 @@
+.checkbox {
+  @apply relative grid items-start gap-2;
+
+  grid-template-columns: min-content 1fr;
+}
+
+.labelSmall {
+  @apply text-sm;
+  --label-line-height: var(--eds-size-line-height-sm);
+}
+
+.labelMedium {
+  @apply text-body;
+  --label-line-height: var(--eds-size-line-height-body);
+}
+
+.label {
+  @apply text-neutral-700;
+
+  /* Shift label so the checkbox always appears vertically centered
+  with the *first* line of text, in case label wraps. */
+  margin-top: calc((var(--checkbox-svg-size) - var(--label-line-height)) / 2);
+}
+
+.inputWrapper {
+  @apply relative;
+}
+
+.checkboxInput {
+  @apply cursor-pointer;
+
+  /* Remove the checkbox from the page flow, positioning it on top of the SVG */
+  position: absolute;
+
+  /* Set same dimensions as the CheckboxSvg element.  */
+  height: var(--checkbox-svg-size);
+  width: var(--checkbox-svg-size);
+
+  /* Hide the input element visually.
+  This ensures the checkbox is still "physically" present so that all users,
+  especially on touch screen readers, still interact with the real checkbox element
+  where it would naturally be present. */
+  opacity: 0;
+}
+
+.checkboxInput + svg {
+  @apply text-neutral-500 align-top inline-block;
```

Comment:
Added `inline-block` to match SvgIcon styles & to override [Tailwind Preflight's ](https://tailwindcss.com/docs/preflight#images-are-block-level) `display: block` styling.

confirmed custom positioning story is as expected:
![Screen Shot 2022-01-14 at 2 58 55 PM](https://user-images.githubusercontent.com/15840841/149577603-c9524ef9-eca1-4bca-ac7e-2a922fa29acd.png)